src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Slider/SliderHandler.Android.cs

```diff
@@ -81,10 +88,15 @@ public static void MapThumbColor(SliderHandler handler, ISlider slider)
 
 		void OnProgressChanged(SeekBar seekBar, int progress, bool fromUser)
 		{
-			if (VirtualView == null)
+			if (VirtualView == null || !fromUser)
 				return;
 
-			VirtualView.Value = progress;
+			var min = VirtualView.Minimum;
+			var max = VirtualView.Maximum;
+
+			var value = min + (max - min) * (progress / 1000.0);
```

Comment:
Can this 1000.0 be a constant? It will get lost and also a nice name will explain.

Reply:
I see the progress PR uses a 10K number. What is a good number for here? What if this slider needs to be a bigger range. For example, I am making an app that wants to show weight in KG for filling a plane. I can't get any higher precision, even if I set my max in the app to 100,000. Maybe this should be higher? Maybe it should be based on the input range? So if I say 1000-50,000, then this is calculate to mean that the slider needs to support a range from 0-500,000, so I could get better precision.

With the current formula, the range 1K-50K basically means that each step is 49. So 1049, 1098, 1147.